tests/kuttl/tests/openstackcontrolplane_scale/06-assert.yaml

```diff
@@ -1,43 +0,0 @@
-#
-# Check for:
-#
-# - OpenStackNet (should be preserved with all historical reservations)
-#
```

Comment:
Is this no longer a valid test?

Reply:
it is! I deleted 06 by accident and only restored 06-delete_openstackcontrolplane.yaml and 06-errors.yaml .. will restore it.

Reply:
done